core/src/main/java/org/infinispan/transaction/impl/AbstractCacheTransaction.java

```diff
@@ -50,6 +51,11 @@ public abstract class AbstractCacheTransaction implements CacheTransaction {
    /** Holds all the keys that were actually locked on the local node. */
    private final AtomicReference<Set<Object>> lockedKeys = new AtomicReference<>();
 
+   /**
+    * Holds all the keys the local node tries to acquire. They might not be locked yet.
+    */
+   private Set<Object> inspectedKeys = null;
```

Comment:
We need this to inspect the LockContainer. I couldn't utilize the `lockedKeys` nor the `affectedKeys` since they are necessary for other functionalities.